contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -925,6 +925,11 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 					size_t startPosition = id->keyword()->TRANSLATE()->getSymbol()->getStartIndex();
 					rewritten_query_fragment.emplace(std::make_pair(startPosition, std::make_pair("", "sys.")));
 				}
+				if (id->keyword()->SUBSTRING())
+				{
+					size_t startPosition = id->keyword()->SUBSTRING()->getSymbol()->getStartIndex();
+					rewritten_query_fragment.emplace(std::make_pair(startPosition, std::make_pair("", "sys.")));
+				}
```

Comment:
Instead of adding explicit for each built-in functions, can't we do something we handle for all of the built-in function?

Reply:
Rohit is working on this in https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2790